tests/step-039.sh

```diff
@@ -12,6 +12,14 @@ mkdir -p /tmp/start/dir && cd /tmp/start/dir || exit 1
 mkdir -p /tmp/empty &&
     GIT_TEMPLATE_DIR=/tmp/empty git init || exit 1
 
+git config --local githooks.single.install "Y"
```

Comment:
I think perhaps leave this test to verify that single installs still work, but perhaps check the deprecation warming was present in the output.
I would then change test 042 to test that the update fails in a single repo install.
What do you think?

Reply:
The  test039.sh of this PR actually tests 
- failing of the install
- single install should still work.

I will add the same in test 042. first checking that the update fails, then  unset the single flag  and
run the update.